src/pages/Main/Components/ProjectCards.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useEffect, useRef } from 'react';
+import ProjectCard from './ProjectCard';
+import Button from '../../../components/Button/Button';
+import { useFilterContext } from '../Main';
+import styled from 'styled-components/macro';
+
+const SORT = [
+  {
+    0: {
+      url: 'Latest',
+      kor: '최신순',
```

Comment:
```suggestion
const SORT = [{
      url: 'Latest',
      kor: '최신순',
```
key가 인덱스인게 어색한데 그냥 배열로 쓰는건 어떨까요?

Reply:
글게요 종택님 리뷰 보니까 굳이 li value로 key 만들어줄 이유가 없었네용?? 수정했습니다!